www/waterfall.inc

```diff
@@ -326,7 +326,11 @@ function _GetMapOrImage($rows, $url, $page_events, $options, $is_image_map, $pag
     $data_width = $width - $data_x - 3;
 
     // Figure out the scale.
+    $min_ms = 0;
     $max_ms = 0;
+    if (@$_REQUEST['min'] > 0 && @$_REQUEST['min'] < @$_REQUEST['max']) {
```

Comment:
Can you address `$max_ms` effective value? As is `min` it only works if a `max` value is provided